e2e/cypress/integration/messaging/emoji_picker_keyboard_usability_spec.js

```diff
@@ -64,11 +64,69 @@ describe('MM-13064 - Emoji picker keyboard usability', () => {
         cy.get('#emojiPickerAliasesPreview').should('have.text', ':smile_cat:');
     });
 
-    xit('If the down arrow key is pressed at any time, move the selection down in the emoji picker');
-    xit('If the up arrow is pressed when the emoji selector is already on the top row of emoji, set the cursor to the beginning of the string. Otherwise, up arrow moves the emoji selector up a row');
-    xit('If CTRL/CMD + Arrow Left/Up is pressed at any time, set the cursor to the beginning of the string when the cursor is at the end of the string');
-    xit('If CTRL/CMD + Arrow Right/Down is pressed at any time, set the cursor to the end of the string when the cursor is at the beginning of the string');
-    xit('If Shift + Arrow Left/Right is pressed at any time, should select or deselect individual characters based on the cursor position and if characters have or have not been selected');
-    xit('If Shift + CTRL/CMD + Arrow Left/Up is pressed at any time, selects the string to the left of the cursor');
-    xit('If Shift + CTRL/CMD + Arrow Right/Down is pressed at any time, selects the string to the right of the cursor');
```

Comment:
Only related tests were  added. Previously skipped tests will be added on separate PR.